setup.sh

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2024, FusionAuth, All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function show_help() {
+  echo -e "\nUsage: $(basename "${0}") [options]\n"
+  echo -e "    -h --help      Show this help message"
+  echo -e "    -u --url       Required, URL of your FusionAuth instance"
+  echo -e "    -k --key       Required, FusionAuth API key"
+  echo -e "    -t --tenant    Optional, FusionAuth Tenant ID"
+  exit
+}
+
+function argerr() {
+  echo -e "\n⛔️ ${1}"
+  exit 1
+}
+
+function parse_args() {
+  while [[ $# -gt 0 ]]; do
+    case "${1}" in
+    -h | --help) show_help ;;
+    -u | --url)
+      shift
+      if (($# < 1)); then argerr "--url requires the URL of your FusionAuth instance"; fi
+      FA_URL="${1:-}"
+      shift
+      ;;
+    -k | --key)
+      shift
+      if (($# < 1)); then argerr "--key requires a valid FusionAuth API key"; fi
+      FA_API_KEY="${1:-}"
+      shift
+      ;;
+    -t | --tenant)
+      shift
+      if (($# < 1)); then argerr "--tenant requires your FusionAuth Default tenant ID"; fi
+      FA_TENANT_ID="${1:-}"
+      shift
+      ;;
+    *) argerr "Unsupported flag '${1}'" ;;
+    esac
+  done
+
+  echo ""
+
+  if [[ -z ${FA_URL:-} ]]; then
+    echo "⛔️ missing required --url param"
+    show_help
+    exit 1
+  else
+    if ! [[ ${FA_URL} == "http"* ]]; then
+      echo "⛔️ URL must be in the format https://your.fusionauth.host"
+      exit 1
+    fi
+    echo "✅ URL = ${FA_URL}"
+  fi
+
+  if [[ -z ${FA_API_KEY:-} ]]; then
+    echo "⛔️ missing required --key param"
+    show_help
+    exit 1
+  else
+    echo "✅ API Key = ${FA_API_KEY}"
+  fi
+
+  if [[ -z ${FA_TENANT_ID:-} ]]; then
+    FA_TENANT_ID="efb21cfc-fa60-46f4-9598-889151e58517"
+    echo "⚠️ --tenant was not set, using default value"
+  fi
+  echo "✅ Tenant ID = ${FA_TENANT_ID}"
+}
+
+function set_json_parser() {
```

Comment:
Following my reasoning of "this is just for us" (not sure if others agree) but I think that it is safe to assume that the environment you are running in is the one created by the dev setup script. So as long as the script has it I think you can assume that utility is present.

Reply:
Does having this check in here hurt anything?